macos/Runner/MainFlutterWindow.swift

```diff
@@ -5,6 +5,14 @@
 import Cocoa
 import FlutterMacOS
 
+func windowOrigin(viewId: Int64) -> CGPoint {
+  let screen = NSScreen.screens[2]
```

Comment:
This line appears to be crashing on my system because I don't have that many screens.

```
2023-05-10 14:38:09.263 mvp[11889:1476610] *** Terminating app due to uncaught exception 'NSRangeException', reason: '*** __boundsFail: index 2 beyond bounds [0 .. 1]'
*** First throw call stack:
(
    0   CoreFoundation                      0x00007ff804e411ba __exceptionPreprocess + 242
    1   libobjc.A.dylib                     0x00007ff80496742b objc_exception_throw + 48
    2   CoreFoundation                      0x00007ff804f1b3ea -[__NSCFString characterAtIndex:].cold.1 + 0
    3   CoreFoundation                      0x00007ff804f229d0 __boundsFail + 70
    4   CoreFoundation                      0x00007ff804d90492 -[__NSArrayI objectAtIndex:] + 38
    5   libswiftCore.dylib                  0x00007ff8142e36a6 $ss12_ArrayBufferV19_getElementSlowPathyyXlSiF + 134
    6   libswiftCore.dylib                  0x00007ff8142e6c09 $sSayxSicig + 153
    7   mvp                                 0x0000000104452ac5 $s3mvp12windowOrigin6viewIdSo7CGPointVs5Int64V_tF + 165
    8   mvp                                 0x000000010445326f $s3mvp17MainFlutterWindowC12awakeFromNibyyF + 623
    9   mvp                                 0x000000010445335c $s3mvp17MainFlutterWindowC12awakeFromNibyyFTo + 28
    10  CoreFoundation                      0x00007ff804dbedca -[NSSet makeObjectsPerformSelector:] + 231
    11  AppKit                              0x00007ff807e39ca1 -[NSIBObjectData nibInstantiateWithOwner:options:topLevelObjects:] + 1367
    12  AppKit                              0x00007ff807e2e49b loadNib + 420
    13  AppKit                              0x00007ff807e2d885 +[NSBundle(NSNibLoading) _loadNibFile:nameTable:options:withZone:ownerBundle:] + 737
    14  AppKit                              0x00007ff807e2d4af -[NSBundle(NSNibLoading) loadNibNamed:owner:topLevelObjects:] + 201
    15  AppKit                              0x00007ff807e2d28d +[NSBundle(NSNibLoading) loadNibNamed:owner:] + 394
    16  AppKit                              0x00007ff807e1fdd7 NSApplicationMain + 566
    17  mvp                                 0x0000000104456da9 main + 9
    18  dyld                                0x00007ff80499441f start + 1903
)
libc++abi: terminating due to uncaught exception of type NSException
```

Reply:
Changing this to the primary screen works for me to avoid the crash:

```suggestion
  let screen = NSScreen.screens[0]
```